docs/content/preview/yugabyte-platform/troubleshoot/universe-issues.md

```diff
@@ -234,6 +244,22 @@ Note that YBA will only collect files if you have sufficient permissions to requ
   - Statefulsets.yaml
   - Persistentvolumeclaims.yaml
 
+#### Prometheus metrics
+
+This component collects Prometheus metrics (Promdump collection) for a given universe. You can select the duration of collection and the metrics to collect from the Support Bundle UI for Prometheus metrics.
+
+A brief description of each metric is as follows:
```

Comment:
The Prometheus metrics for the selected universe. You can customize the timeframe and the metrics to collect.

You can include the following metrics with the support bundle:

- Master Export - YB-Master metrics.
- Node Export - `node_exporter` OS level metrics.
- Platform - YugabyteDB Anywhere metrics.
- Prometheus - YugabyteDB Anywhere Prometheus metrics.
- TServer Export - YB-TServer metrics; these are scraped from the TServer processes, and mostly cover the storage layer.
- YSQL Export - YSQL query layer metrics.
- YCQL Export - YCQL query layer metrics.

The Prometheus metrics are stored in JSON files in the `support_bundle/YBA/promdump` directory.